front/src/pages/companies/CompanyShow.tsx

```diff
@@ -0,0 +1,205 @@
+import { PropertyBoxItem } from '@/ui/components/property-box/PropertyBoxItem';
+import { IconBuildingSkyscraper, IconLink, IconMap } from '@/ui/icons/index';
+import { WithTopBarContainer } from '@/ui/layout/containers/WithTopBarContainer';
+import {
+  beautifyExactDate,
+  beautifyPastDateRelativeToNow,
+} from '@/utils/datetime/date-utils';
+import { getLogoUrlFromDomainName } from '@/utils/utils';
+import { CommentableType } from '~/generated/graphql';
+
+const StyledCompanyContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  background: ${({ theme }) => theme.background.primary};
+  border: 1px solid ${({ theme }) => theme.border.color.medium};
+  border-radius: 8px;
+  display: flex;
+  flex: 1 0 0;
+`;
+
+const StyledLeftPanelContainer = styled.div`
+  align-items: flex-start;
+  align-self: stretch;
+  background: ${({ theme }) => theme.background.secondary};
+  border-radius: 8px;
+  border-right: 1px solid ${({ theme }) => theme.border.color.light};
+  padding: 0px ${({ theme }) => theme.spacing(3)};
+  width: 320px;
+`;
+
+const StyledRightPanelContainer = styled.div`
+  align-items: flex-start;
+  align-self: stretch;
+  display: flex;
+  flex: 1 0 0;
+  flex-direction: column;
+  justify-content: center;
+  overflow: hidden;
+`;
+
+const StyledPropertyBoxContainer = styled.div`
+  padding: 0px 12px;
+`;
+
+export function CompanyShow() {
+  const companyId = useParams().companyId ?? '';
+
+  const { data } = useCompanyQuery(companyId);
+  const company = data?.findUniqueCompany;
+
+  const theme = useTheme();
+
+  return (
+    <WithTopBarContainer
+      title={company?.name ?? ''}
+      icon={<IconBuildingSkyscraper size={theme.icon.size.md} />}
+    >
+      <StyledCompanyContainer>
+        <StyledLeftPanelContainer>
+          <TopLeftHeaderOnShowPage
+            logo={getLogoUrlFromDomainName(company?.domainName ?? '')}
+            title={company?.name ?? ''}
+            date={company?.createdAt ?? ''}
+          />
+          <StyledPropertyBoxContainer>
+            <PropertyBox>
+              <>
+                <PropertyBoxItem
+                  icon={<IconLink />}
+                  value={company?.domainName ?? ''}
+                  link={
+                    company?.domainName ? 'https://' + company?.domainName : ''
+                  }
+                />
+                <PropertyBoxItem
+                  icon={<IconMap />}
+                  value={company?.address ? company?.address : 'No address'}
+                />
+              </>
+            </PropertyBox>
+          </StyledPropertyBoxContainer>
+        </StyledLeftPanelContainer>
+        <StyledRightPanelContainer>
+          <Timeline
+            entity={{ id: company?.id ?? '', type: CommentableType.Company }}
+          />
+        </StyledRightPanelContainer>
+      </StyledCompanyContainer>
+    </WithTopBarContainer>
+  );
+}
+
+///////////////////////////////////////////////////////////////
```

Comment:
what happened here? :D

Reply:
Haha yes I usually do all components in a single file to iterate fast then only split after 😌